mono/metadata/w32file-unix.c

```diff
@@ -2348,11 +2466,15 @@ mono_w32file_create(const gunichar2 *name, guint32 fileaccess, guint32 sharemode
 		handle_type = MONO_W32HANDLE_FILE;
 	}
 
+	MONO_ENTER_GC_SAFE; /* FIXME: mono_w32handle_new_fd should be updated with coop transitions */
 	handle = mono_w32handle_new_fd (handle_type, fd, &file_handle);
```

Comment:
w32handle are not coop aware yet. We need to transition all the w32socket, w32process, etc. to be coop aware before being able to transition w32handle.